gr-data.csv

```diff
@@ -51,6 +51,10 @@ https://github.com/romalev/reactive-spring-web-flux-service,19e64d356840720f78b5
 https://github.com/romalev/reactive-spring-web-flux-service,19e64d356840720f78b50bd4c08cf5a1d7391834,.,com.reactive.service.ApplicationTester.verifyMultiplyOperationOnSuccess,ID,,,please run this project using NonDex v1.2
 https://github.com/romalev/reactive-spring-web-flux-service,19e64d356840720f78b50bd4c08cf5a1d7391834,.,com.reactive.service.ApplicationTester.verifyPowerOf2SubSqrtOperationOnSuccess,ID,,,please run this project using NonDex v1.2
 https://github.com/romalev/reactive-spring-web-flux-service,19e64d356840720f78b50bd4c08cf5a1d7391834,.,com.reactive.service.ApplicationTester.verifySubOperationOnSuccess,ID,,,please run this project using NonDex v1.2
+https://github.com/spring-projects/spring-boot,daa3d457b71896a758995c264977bdd1414ee4d4,spring-boot-project/spring-boot-actuator-autoconfigure,org.springframework.boot.actuate.autoconfigure.endpoint.web.documentation.BeansEndpointDocumentationTests.beans,ID,,,
```

Comment:
Put here the commit SHA that you actually ran, not the old commit from when the project was using Maven. Are the modules correct? Are the test names correct?